<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsBrowserMail</class>
 <widget class="QWidget" name="SettingsBrowserMail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>330</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="m_tabBrowserProxy">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tabNetwork">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbIgnoreAllCookies">
         <property name="text">
          <string>Do not accept any incoming cookies</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbEnableHttp2">
         <property name="text">
          <string>Enable HTTP/2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbEnableApiServer">
         <property name="text">
          <string>Enable API server</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="m_txtUserAgent"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Custom User-Agent</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabExternalBrowser">
      <attribute name="title">
       <string>External web browser</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_checkOpenLinksInExternal">
         <property name="text">
          <string>Always open hyperlinks in external web browser</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="m_grpCustomExternalBrowser">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If unchecked, then default system-wide web browser is used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Use custom external web browser</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_12">
          <item row="0" column="0">
           <widget class="QLabel" name="m_lblExternalBrowserExecutable">
            <property name="text">
             <string>Web browser executable</string>
            </property>
            <property name="buddy">
             <cstring>m_txtExternalBrowserExecutable</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="BaseLineEdit" name="m_txtExternalBrowserExecutable">
              <property name="placeholderText">
               <string>Executable file of web browser</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnExternalBrowserExecutable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&amp;Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_lblExternalBrowserArguments">
            <property name="text">
             <string>Parameters</string>
            </property>
            <property name="buddy">
             <cstring>m_txtExternalBrowserArguments</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="BaseLineEdit" name="m_txtExternalBrowserArguments">
            <property name="placeholderText">
             <string>Parameters passed to executable</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_lblExternalBrowserPreset">
            <property name="text">
             <string>Use sample arguments for</string>
            </property>
            <property name="buddy">
             <cstring>m_cmbExternalBrowserPreset</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="m_cmbExternalBrowserPreset">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>Select browser</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="HelpSpoiler" name="m_lblExternalBrowserInfo" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabExternalMail">
      <attribute name="title">
       <string>External e-mail client</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_22">
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="m_grpCustomExternalEmail">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If unchecked, then default system-wide web browser is used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Use custom external e-mail client</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="_2">
          <item row="0" column="0">
           <widget class="QLabel" name="m_lblExternalEmailExecutable">
            <property name="text">
             <string>E-mail client executable</string>
            </property>
            <property name="buddy">
             <cstring>m_txtExternalEmailExecutable</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="BaseLineEdit" name="m_txtExternalEmailExecutable">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Executable file of e-mail client</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnExternalEmailExecutable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&amp;Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_lblExternalEmailArguments">
            <property name="text">
             <string>Parameters</string>
            </property>
            <property name="buddy">
             <cstring>m_txtExternalEmailArguments</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="BaseLineEdit" name="m_txtExternalEmailArguments">
            <property name="placeholderText">
             <string>Parameters passed to executable</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_lblExternalEmailPreset">
            <property name="text">
             <string>Use sample arguments for</string>
            </property>
            <property name="buddy">
             <cstring>m_cmbExternalEmailPreset</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="m_cmbExternalEmailPreset">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>Select client</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="HelpSpoiler" name="m_lblExternalEmailInfo" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabExternalTools">
      <attribute name="title">
       <string>External tools</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QTreeWidget" name="m_listTools">
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_btnAddTool">
           <property name="text">
            <string>&amp;Add tool</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnEditTool">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Edit selected tool</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnDeleteTool">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Delete selected tool</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="HelpSpoiler" name="m_lblToolInfo" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BaseLineEdit</class>
   <extends>QLineEdit</extends>
   <header>baselineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_tabBrowserProxy</tabstop>
  <tabstop>m_grpCustomExternalBrowser</tabstop>
  <tabstop>m_txtExternalBrowserExecutable</tabstop>
  <tabstop>m_btnExternalBrowserExecutable</tabstop>
  <tabstop>m_txtExternalBrowserArguments</tabstop>
  <tabstop>m_cmbExternalBrowserPreset</tabstop>
  <tabstop>m_checkOpenLinksInExternal</tabstop>
  <tabstop>m_grpCustomExternalEmail</tabstop>
  <tabstop>m_txtExternalEmailExecutable</tabstop>
  <tabstop>m_btnExternalEmailExecutable</tabstop>
  <tabstop>m_txtExternalEmailArguments</tabstop>
  <tabstop>m_cmbExternalEmailPreset</tabstop>
  <tabstop>m_listTools</tabstop>
  <tabstop>m_btnAddTool</tabstop>
  <tabstop>m_btnEditTool</tabstop>
  <tabstop>m_btnDeleteTool</tabstop>
  <tabstop>m_cbIgnoreAllCookies</tabstop>
  <tabstop>m_cbEnableHttp2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
